src/Api/Models/Entity/Annotation/AnnotationTagInstance.cs

```diff
@@ -28,8 +36,12 @@ public class AnnotationTagInstanceMap
         {
             public AnnotationTagInstanceMap(EntityTypeBuilder<AnnotationTagInstance> entityBuilder)
             {
-                entityBuilder.HasOne(tag => tag.TagModel).WithMany(model => model.TagInstances)
-                    .HasForeignKey(tag => tag.TagModelId).OnDelete(DeleteBehavior.Cascade);
+                entityBuilder.HasOne(tag => tag.TagModel)
+                    .WithMany(model => model.TagInstances)
+                    .HasForeignKey(tag => tag.TagModelId)
+                    .OnDelete(DeleteBehavior.Cascade);
+                entityBuilder.HasOne(tag => tag.InDocument).WithMany()
```

Comment:
HasForeignKey is missing

entityBuilder.HasOne(tag => tag.InDocument).WithMany()**.HasForeignKey(tag => tag.InDocumentId)**.OnDelete(DeleteBehavior.Cascade)